tap_zendesk/streams.py

```diff
@@ -16,6 +15,12 @@
 LOGGER = singer.get_logger()
 KEY_PROPERTIES = ['id']
 
+REQUEST_TIMEOUT = 300
+TICKET_START_TIME = 1610368140
```

Comment:
@prijendev Why to keep a static value. You can use start time provided in the config.json file. Correct?

Reply:
Yes, updated and removed static time